oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/MetricsQueryService.java

```diff
@@ -50,7 +51,7 @@ private IMetricsQueryDAO getMetricQueryDAO() {
     /**
      * Read metrics single value in the duration of required metrics
      */
-    public long readMetricsValue(MetricsCondition condition, Duration duration) throws IOException {
+    public NullableValue readMetricsValue(MetricsCondition condition, Duration duration) throws IOException {
```

Comment:
I think we don't change this API. Somehow we don't tested this?

We just added a new API --> **readNullableMetricsValue**